lstchain/irf/hdu_table.py

```diff
@@ -260,12 +260,12 @@ def create_event_list(data, run_number, source_name):
     time = Time(data["dragon_time"], format="unix", scale="utc")
     date_obs = time[0].to_value("iso", "date")
 
-    #t_eff, t_elapsed = get_effective_time(data)
-    obs_time = t_stop - t_start
-    deltaT = np.diff(data["dragon_time"].value)
-    deltaT = deltaT[(deltaT > 0) & (deltaT < 0.002)]
-    rate = 1 / np.mean(deltaT)
-    dead_time = np.amin(deltaT)
+    t_eff, t_elapsed = get_effective_time(data)
+    #obs_time = t_stop - t_start
```

Comment:
maybe let's remove these commented lines, otherwise it could be misleading

Reply:
Yes, sorry, I forgot to remove them.

Reply:
This is also from an older commit. It has been resolved in the latest commit.